src/applications/find-forms/widgets/createFindVaFormsPDFDownloadHelper/DownloadHandler.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
```

Comment:
This code is largely the same as what was here before (in `DownloadPDFGuidance`). I just renamed it.